src/draw_parameters/mod.rs

```diff
@@ -378,6 +379,12 @@ pub struct DrawParameters<'a> {
     /// Since this is purely an optimization, this parameter is ignored if the backend doesn't
     /// support it.
     pub primitive_bounding_box: (Range<f32>, Range<f32>, Range<f32>, Range<f32>),
+    
+    /// If enabled, will split the index buffer (if any is used in the draw call) 
+    /// at the MAX value of the IndexType (u8::MAX, u16::MAX or u32::MAX) start a new primitive
+    /// of the same type ("primitive restarting"). Supported on > OpenGL 3.1 or OpenGL ES 3.0 - 
+    /// otherwise, enabling this has no effect.
```

Comment:
If this is not supported, a `FixedIndexRestartingNotSupported` error is returned (which is good). So the last sentence here is wrong.